src/main/java/com/sap/sgs/phosphor/fosstars/data/github/UsesSignedCommits.java

```diff
@@ -27,85 +24,71 @@
 
   /**
-   * The threshold in % of number of verified signed commits against total number of commits in a
+   * The threshold in % of number of signed commits against total number of commits in a
    * year.
    */
   private static final double SIGNED_COMMIT_PERCENTAGE_THRESHOLD = 90;
 
   /**
    * Initializes a data provider.
    *
-   * @param github An interface to the GitHub API.
+   * @param fetcher An interface to GitHub.
    */
-  public UsesSignedCommits(GitHub github) {
-    super(github);
+  public UsesSignedCommits(GitHubDataFetcher fetcher) {
+    super(fetcher);
   }
 
   @Override
   protected Feature supportedFeature() {
-    return USES_VERIFIED_SIGNED_COMMITS;
+    return USES_SIGNED_COMMITS;
   }
 
   @Override
   protected Value fetchValueFor(GitHubProject project) throws IOException {
-    logger.info("Figuring out if the project uses verified signed commits ...");
+    logger.info("Figuring out if the project uses signed commits ...");
     return usesSignedCommits(project);
   }
 
   /**
    * It gathers the list of commits within one year from the current date. Then it calculates
-   * density ratio. The density ratio is the percentage calculation of number of verified signed
+   * density ratio. The density ratio is the percentage calculation of number of signed
    * commits against the total number of commits in the year. If the density ratio in % crosses a
    * certain percentage threshold ({@link #SIGNED_COMMIT_PERCENTAGE_THRESHOLD}), then it says that
-   * the project uses verified signed commits. Otherwise, it says that the project does not use
-   * verified signed commits.
+   * the project uses signed commits. Otherwise, it says that the project does not use
+   * signed commits.
    *
    * @param project The project.
    * @return {@link BooleanValue} value of the feature.
    */
   private Value<Boolean> usesSignedCommits(GitHubProject project) {
     try {
-      return new BooleanValue(USES_VERIFIED_SIGNED_COMMITS, askGithub(project));
+      return new BooleanValue(USES_SIGNED_COMMITS, askGithub(project));
     } catch (IOException e) {
       logger.warn("Couldn't fetch data from api.github.com!", e);
-      return UnknownValue.of(USES_VERIFIED_SIGNED_COMMITS);
+      return UnknownValue.of(USES_SIGNED_COMMITS);
     }
   }
 
   /**
-   * Check if a project uses signed verified commits.
+   * Check if a project uses signed commits.
    *
    * @param project The project.
-   * @return boolean True if the project uses verified signed commits. Otherwise, False
+   * @return boolean True if the project uses signed commits. Otherwise, False
    * @throws IOException caused during REST call to GitHub API.
    */  
   private boolean askGithub(GitHubProject project) throws IOException {
-    List<GHCommit> yearCommits = gitHubDataFetcher().commitsAfter(ONE_YEAR_AGO, project, github);
+    List<Commit> yearCommits = fetcher.localRepositoryFor(project).commitsAfter(ONE_YEAR);
```

Comment:
Now I think about it, maybe we should change the name `commitsAfter()` -> `commitsWithin()`

Reply:
Let me find a better name for the constant instead.

Reply:
ok

Reply:
Actually, that's a good point to add a new `commitsWithin()` method but it should take a `Duration`.

Reply:
Just added the method.